eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -167,6 +167,12 @@
           "new": "method byte\\[\\] com\\.azure\\.security\\.keyvault\\.keys\\.models\\.JsonWebKey::(g|s)et(D|Dp|Dq|E|K|N|P|Q|Qi|T|X|Y)\\(\\)",
           "justification": "The class used by the serialization/deserialization annotations is package private and moving it is not a breaking change."
         },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.(added|attributeValueChanged|attributeAdded)",
+          "new": "(class|method void|parameter void) com\\.azure\\.search\\.documents\\.indexes\\.models\\..*",
+          "justification": "Generated classes were moved into public API, these annotations were already being used in implementation used during serialization and deserialization."
+        },
```

Comment:
Was this revapi suppression intentional and was this something that had to be done to enable Java 17?